### DISCUSSION TITLE
Fast & Slow action buttons for players in the combat tracker

### DISCUSSION THREAD

**Discussion by giryan** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#214] [Ideas]
I really loved the Fast & Slow actions that were added to the combat tracker, and so happily switched over to them.
However I found that my players couldn't see or manipulate them, and so I think it would be good to  add the two buttons if owner as well.

I made a change locally, and will PR and do things properly if it's considered worthwhile.
https://github.com/giryan/forbidden-lands-foundry-vtt/commit/b7a16c687287733cb17107a99d102a49579e1e9d
I've not yet entirely got things set up and haven't got husky running etc.. (I just changed the code and ran it :) ) so I'll make sure that all of that is done before actually PRing.

**Comment by aMediocreDad** [comment]
Hey! This was an oversight! I'll updating the templates with the correct behaviour.

**Reply by giryan** [reply]
Great! Thanks :D
